modular_skyrat/modules/modular_implants/code/nif_persistence.dm

```diff
@@ -74,6 +81,16 @@
 	new_nif.current_theme = persistence.nif_theme
 	new_nif.is_calibrated = persistence.nif_is_calibrated
 	new_nif.rewards_points = persistence.stored_rewards_points
+
+	var/list/persistent_nifsoft_paths = list()
+	for(var/text as anything in splittext(persistence.persistent_nifsofts, "&"))
```

Comment:
You don't cast the type of your variable, the `as anything` serves no purpose here.
```suggestion
	for(var/text in splittext(persistence.persistent_nifsofts, "&"))
```